docs/source/configuration.rst

```diff
@@ -335,6 +335,19 @@ This relationship can further be maintained in the JSON-LD structured data using
             ssn: "http://www.w3.org/ns/ssn/"
             Datastream: sosa:isMemberOf
 
+Sometimes, the JSON-LD desired is more complicated than can be achieved by simply aliasing properties using a
```

Comment:
We should articulate that this functionality is triggered for a single feature collection item.